spec/dummy_app/app/models/post_with_status.rb

```diff
@@ -5,5 +5,9 @@
 class PostWithStatus < ApplicationRecord
   has_paper_trail
 
-  enum status: { draft: 0, published: 1, archived: 2 }
+  if ActiveRecord::VERSION::MAJOR >= 8
```

Comment:
```suggestion
  if ActiveRecord::VERSION::MAJOR >= 7
```

I think the new syntax was added in 7.0 so this would enable removing the condition sooner (when 6.1 is dropped)

Reply:
Good idea! 👍